tools/cmake/mbed_greentea.cmake

```diff
@@ -35,26 +35,23 @@ macro(mbed_greentea_add_test)
         ${ARGN}
     )
 
-    set(TEST_NAME ${MBED_GREENTEA_TEST_NAME})
-
     add_subdirectory(${MBED_PATH} build)
 
-    add_executable(${TEST_NAME})
+    add_executable(${MBED_GREENTEA_TEST_NAME})
```

Comment:
The "MBED_GREENTEA_TEST_NAME" change should also go in its own commit.

Reply:
Done 👍